src/server/string_family.cc

```diff
@@ -379,31 +379,27 @@ OpStatus OpMSet(const OpArgs& op_args, const ShardArgs& args) {
   return result;
 }
 
-// emission_interval_ms assumed to be positive
+// emission_interval_ns assumed to be positive
 // limit is assumed to be positive
 OpResult<array<int64_t, 5>> OpThrottle(const OpArgs& op_args, const string_view key,
-                                       const int64_t limit, const int64_t emission_interval_ms,
+                                       const int64_t limit, const int64_t emission_interval_ns,
```

Comment:
It would be nice to use `std::chrono::nanoseconds/milliseconds` in this function to avoid accidentally mixing up time units, but we do many checks based on `INT64` limits which will cause problems if we use a real time duration unit.